csfieldguide/static/interactives/no-help/scss/no-help.scss

```diff
@@ -0,0 +1,6 @@
+#interactive-no-help {
+    background-color: #4caf50;
+    border-color: #4caf50;
+    color: #fff;
+    margin-bottom: 14px;
```

Comment:
Instead of specifying an exact margin, you can use Bootstrap margin helpers to add the spacing.
See: https://getbootstrap.com/docs/4.1/utilities/spacing/
They are very useful once you start using them. In this case I would remove this style rule and add `mb-3` onto the `a` element.